src/lib/stellar/transactions.js

```diff
@@ -1,6 +1,6 @@
-import { TransactionBuilder, Networks, Server, Operation, Asset, Memo } from 'stellar-sdk'
+import { TransactionBuilder, Networks, Operation, Asset, Memo, Contract, xdr, Address, StrKey, SorobanRpc } from 'stellar-sdk'
```

Comment:
The `rpc` module is the preferred import method for the rpc server from the SDK
```suggestion
import { TransactionBuilder, Networks, Operation, Asset, Memo, Contract, xdr, Address, StrKey, rpc } from '@stellar/stellar-sdk'
```